src/Symfony/Bundle/FrameworkBundle/Application/TranslationTrait.php

```diff
@@ -0,0 +1,64 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Bundle\FrameworkBundle\Application;
+
+/**
+ * Translation trait.
+ *
+ * @author Fabien Potencier <fabien@symfony.com>
+ * @author Michaël Garrez <michael.garrez@gmail.com>
+ */
+trait TranslationTrait
```

Comment:
I did use the same name as the Silex TranslationTrait, don't you think we should use the same pattern?